app/collectors/securityGroup.scala

```diff
@@ -21,46 +19,47 @@ class SecurityGroupCollectorSet(accounts: Accounts, prismController: Prism) exte
 case class AWSSecurityGroupCollector(origin:AmazonOrigin, resource:ResourceType, prismController: Prism, crawlRate: CrawlRate)
     extends Collector[SecurityGroup] {
 
-  def fromAWS( secGroup: AWSSecurityGroup, lookup:Map[String,SecurityGroup]): SecurityGroup = {
+  def fromAWS( secGroup: AwsSecurityGroup, lookup:Map[String,SecurityGroup]): SecurityGroup = {
     def groupRefs(rule: IpPermission): Seq[SecurityGroupRef] = {
-      rule.getUserIdGroupPairs.asScala.map { pair =>
-        SecurityGroupRef(pair.getGroupId, pair.getUserId, lookup.get(pair.getGroupId).map(_.arn))
+      rule.userIdGroupPairs.asScala.map { pair =>
+        SecurityGroupRef(pair.groupId, pair.userId, lookup.get(pair.groupId).map(_.arn))
       }
     }.toSeq
 
-    val rules = secGroup.getIpPermissions.asScala.map { rule =>
+    val rules = secGroup.ipPermissions.asScala.map { rule =>
       Rule(
-        rule.getIpProtocol.replace("-1","all"),
-        Option(rule.getFromPort).map(_.toInt),
-        Option(rule.getToPort).map(_.toInt),
-        rule.getIpv4Ranges.asScala.toSeq.map(_.toString).sorted.wrap,
-        rule.getIpv6Ranges.asScala.toSeq.map(_.toString).sorted.wrap,
+        rule.ipProtocol.replace("-1","all"),
+        Option(rule.fromPort).map(_.toInt),
+        Option(rule.toPort).map(_.toInt),
+        rule.ipRanges.asScala.toSeq.map(_.toString).sorted.wrap,
+        rule.ipv6Ranges.asScala.toSeq.map(_.toString).sorted.wrap,
```

Comment:
I think that this is the best way of solving this - the IP range object also includes a description which we previously didn't deal with.
```suggestion
        rule.ipRanges.asScala.toSeq.map(_.cidrIp).sorted.wrap,
        rule.ipv6Ranges.asScala.toSeq.map(_.cidrIpv6).sorted.wrap,
```

Reply:
This is looking a lot better now:
```
<           "{CidrIp: xx.xx.xxx.x/xx,}"
---
>           "xx.xx.xxx.x/xx"
```